docs/README.md

```diff
@@ -0,0 +1,9 @@
+Some `*.md` files here are generated from their `*.template.md` counterpart. Generate them with:
+
+```bash
+$ npm install
+$ node build.js
```

Comment:
You make me realize that the `build.js` script isn't an ideal way of running the docs build. I'll improve this and keep you updated

Reply:
You can now `npm run docs` https://github.com/reframejs/reframe/commit/3f762384aa434729dbf726893f1c3cca1bfe6039

Reply:
(I changed the docs readme accordingly https://github.com/reframejs/reframe/commit/7d6847d1c59113230675a31be405a546eebbe69a)